### PR TITLE
Share port support

### PR DESCRIPTION
**Pull request by DisDis**

### CODE REVIEW

**Comment by DisDis** [comment]
@sigurdm ping

File: lib/src/server/server.dart [hunk 1]
```diff
@@ -99,7 +102,7 @@ class Server {
       server = _secureServer;
```
Comment (sigurdm) [inline lib/src/server/server.dart]: You need to also pass the parameters in the secure case

Comment (DisDis) [reply lib/src/server/server.dart]: done

**Comment by DisDis** [comment]
@sigurdm please merge this PR

**Comment by DisDis** [comment]
@sigurdm ping

**Pull request merged**